packages/react-reconciler/src/ReactFiberHooks.old.js

```diff
@@ -478,11 +521,25 @@ export function renderWithHooks<Props, SecondArg>(
   ReactCurrentDispatcher.current = ContextOnlyDispatcher;
 
   if (__DEV__) {
+    if (
```

Comment:
Calling this out - this isn't pretty but it adds the warning for going from some hooks to no hooks.